server/events/vcs/common/common.go

```diff
@@ -3,12 +3,14 @@
 package common
 
 import (
+	"fmt"
 	"math"
 )
 
-// AutomergeCommitMsg is the commit message Atlantis will use when automatically
-// merging pull requests.
-const AutomergeCommitMsg = "[Atlantis] Automatically merging after successful apply"
+// AutomergeCommitMsg returns the commit message to use when automerging.
+func AutomergeCommitMsg(pullNum int) string {
```

Comment:
Can we create a test for this function?

Reply:
I can, yeah. I don't understand why we would write tests for, basically, "does fmt.Sprintf work the way we think it does?". Any chance you can expand on the goals we'd be wanting to achieve by testing this?

Reply:
What if the function was modified to include more logic to change the text on some condition? If so, the person who contributed the change would more likely be willing to edit an existing test than to create a new test. If this test isn't created now, it may never be.

I think every pr should strive to not decrease coverage as a bare minimum requirement

Reply:
Gotcha, yeah, that make sense :) Thanks for the "behind the scenes" explanation :)